nginx/error_pages/400.html

```diff
@@ -1,16 +1,20 @@
 <html lang="en">
 <head>
-    <title>403</title>
+    <title><!--#echo var="status" --></title>
     <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
     <link rel="preconnect" href="https://fonts.googleapis.com">
     <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
     <link href="https://fonts.googleapis.com/css2?family=Inter:wght@400;600;800&family=Manrope:wght@200;300;400;500;600;700;800&family=Roboto:wght@100&display=swap" rel="stylesheet">
     <style>
-        html, body {
+        html,
+        body {
```

Comment:
It can be in one line.

Reply:
this is the default prettier setting, there is no custom setting in the project.

Reply:
Ok, I will fix it on my own.